File: cpp/HelloWorld.c
// https://www3.ntu.edu.sg/home/ehchua/programming/cpp/gcc_make.html
// https://bytes.com/topic/c/answers/646010-self-deleting-exe

#include <stdio.h>
 
int main(int argc, char* argv[]) {
    printf("Hello, world!\n");
    remove(argv[0]);        // Self deleting for Linux only
    return 0;
}

File: kotlin/HelloWorld.kt

fun main() {
    print("Hello ")
    print("world!")
    println("Hello world!")
    var a:String?="2"
    var b:String?="1"
    val c = if(a == null || b == null)a?:b else "e"
    print(c)
}
// fun main(args: Array<String>) {
//     println(args.contentToString())
// }